src/lib/offline-api/index.ts

```diff
@@ -351,6 +413,33 @@ class OfflineAPI {
     return entries
   }
 
+  async getLinks (childId: string, locales: string[]): Promise<Link[]> {
+
+    const links: Link[] = []
+
+    this.entries.forEach((entry) => {
```

Comment:
Code style wise, I think a `for ... of` loop would be preferable here.

Reply:
Done. I left the innermost forEach as is since iteration over entries() to get both the key and value is slower that using forEach and there are potentially a lot of loop iterations, so it might be noticeable.